h/static/scripts/helpers/time-helpers.coffee

```diff
@@ -14,6 +14,7 @@ BREAKPOINTS = [
   [2 * day,    'yesterday',      day   ]
   [month,      '{} days ago',    day   ]
   [year,       '{} months ago',  month ]
+  [year*2,     'one year ago',   year  ]
```

Comment:
You're going to hate me, but everything else in here is `2 * foo`, not `foo*2`. My compulsion for consistency won't let me merge this.